main.ledger

```diff
@@ -1545,12 +1580,26 @@
     Liabilities:Reimbursement:Zach Latta
     ; Receipt: a2875e2a6987e90e4817b121274cfdf6.pdf
      
+2016/02/03 Someone didn't cash their check
+    Assets:Bank:Checking                              $1900.00
+    Expenses:Salary
+    ; Receipt: 9b1b2f9c1dcc2d6a19757f88fa422fcb.png
```

Comment:
Man... my neurons did not make the connection about the receipt signifying
the correct amount. Sorry about this.

On Wed, Apr 13, 2016 at 11:36 PM, JESSICA <notifications@github.com> wrote:

> In main.ledger
> <https://github.com/hackclub/finances/pull/12#discussion_r59668699>:
>
> > @@ -1545,12 +1580,26 @@
> >      Liabilities:Reimbursement:Zach Latta
> >      ; Receipt: a2875e2a6987e90e4817b121274cfdf6.pdf
> >
> > +2016/02/03 Someone didn't cash their check
> > +    Assets:Bank:Checking                              $1900.00
> > +    Expenses:Salary
> > +    ; Receipt: 9b1b2f9c1dcc2d6a19757f88fa422fcb.png
>
> Y U ASKIN ME IF I AM SURE! I included the receipt!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/finances/pull/12/files/050535646cac2b1339431fbe861ff24075f4fdb7#r59668699>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>